contrib/TRAMPOLINE

```diff
@@ -9,6 +9,11 @@
                   Mawww's experiment for a better code editor
 +=----------------------------------------------------------------------------=+
 
+To view this inside Kakoune, map it to some key like so:
```

Comment:
I am not sure this is the best place to put that advice, and in general would avoid suggesting to pull files from the web.

Reply:
> I am not sure this is the best place to put that advice,

What would be a better place to put this? If it would resolve #1122

> and in general would avoid suggesting to pull files from the web.

What's wrong with pulling files from the web? If downloading repeatedly is a problem the user can save the `TRAMPOLINE` file somewhere

Reply:
> What would be a better place to put this?

As @Screwtapello said, move it somewhere inside `rc` or `doc`, so it will be installed by default